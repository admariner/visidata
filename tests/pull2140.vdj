#!vd -p
{"sheet": null, "col": null, "row": null, "longname": "open-file", "input": "sample_data/empty-table.html", "keystrokes": "o", "comment": null}
