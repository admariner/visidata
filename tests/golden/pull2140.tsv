name	rows	cols	id	classes	title	aria_label	caption	summary	heading
table_0	0	0		test_empty					
links	0	5							
